packages/pods-core/src/builtin/GithubIssuePod.ts

```diff
@@ -395,6 +395,19 @@ type GithubIssuePublishPodCustomOpts = {
    * if set to false, starts a discussion without the contents of note body
    */
   includeNoteBodyInDiscussion?: boolean;
+  /**
+   * aliasMapping for frontmatter
+   */
+  aliasMapping?: AliasMapping;
+};
+
+type AliasMapping = {
+  assignees: AliasMappingLvl2;
+  status: AliasMappingLvl2;
+};
+type AliasMappingLvl2 = {
+  value?: { [key: string]: string };
```

Comment:
I'm not quite following why this must be a JSON object / dictionary.  

Would you mind including a docs change w/ an example on how to set `aliasMapping`?